src/network-interface.js

```diff
@@ -16,9 +16,13 @@ export class UploadHTTPFetchNetworkInterface extends HTTPFetchNetworkInterface {
         // Build the form
         const formData = new FormData()
         formData.append('operations', JSON.stringify(operation))
-        files.forEach(({ variablesPath, file }) =>
-          formData.append(variablesPath, file)
-        )
+        files.forEach(({ variablesPath, file }) => {
```

Comment:
Believe it's just the path of the file being uploaded, but it's not used server side.